helpers/isRoomAvailable.js

```diff
@@ -0,0 +1,27 @@
+const Reservation = require('../models/reservation')
```

Comment:
I think it would be better to have such function in e.g. hotelsService file.

Reply:
We use it in Two Services as far as i remember

Reply:
Yes, I know, but nothing stop you to import it from hotelsService module

Reply:
Ok